protocol/linkpreview/linkpreview.go

```diff
@@ -130,7 +130,7 @@ type OEmbedUnfurler struct {
 
 type OEmbedResponse struct {
 	Title        string `json:"title"`
-	ThumbnailURL string `json:"thumbnail_url"`
```

Comment:
The rename seems unnecessary, in fact the struct tags here are unnecessary because `OEmbedResponse` is not encoded as JSON anywhere.

Reply:
I think some endpoints return JSON, Like giphy.
That's why the rename fixed the giphy issue.